cloudhub-2/modules/ROOT/pages/ch2-understand-app-restart.adoc

```diff
@@ -0,0 +1,35 @@
+= Understanding App Automatic Restart 
```

Comment:
Should this be "Automatic App Restart"?

Reply:
I think we should avoid the "Understanding" and just call it Automatic App Restarts